### DISCUSSION TITLE
Is there any way to give spaces among the diagrams to align to look good.

### DISCUSSION THREAD

**Discussion by animesh-11** [mingrammer/diagrams#840] [General]
I am trying to add spaces as well as want to make the line a little longer to align the diagrams. Please let me know how to do it ??

from diagrams import Diagram, Cluster
from diagrams.programming.flowchart import Action
from diagrams.programming.flowchart import Inspection
from diagrams.onprem.client import User
from diagrams.onprem.client import Client
#from diagrams.aws.engagement import SimpleEmailServiceSesEmail 

with Diagram("How to order Pizza") as diag:
    #B3 = User("Pizza Received")

    with Cluster("Shop"):
            A1 = User("Order Received")
            A2 = Action("Bake Pizza")
            A3 = Action("Add toping to Pizza")
            A4 = Action("Deliver Pizza")
            A5 = Client("Payment Received")
            A1 >> A2 >> A3  >> A4 >> A5
            
        
    with Cluster("Client"):
        B1 = Inspection("Start")
        B2 = Action("Order Pizza")
        B3 = User("Pizza Received")
        B4 = Action("Make Payment")
        B5 = Inspection("Stop")
        B1 >> B2 >> B3 >> B4 >> B5
        
        B2 >> A1
        A4 >> B3
        B4 >> A5
    
 diag

**Comment by clayms** [comment]
Have a look at the following:
https://github.com/mingrammer/diagrams/issues/364#issuecomment-719957643

https://github.com/mingrammer/diagrams/issues/185#issuecomment-722381623

https://github.com/mingrammer/diagrams/issues/579#issuecomment-899830574
https://github.com/mingrammer/diagrams/issues/579#issuecomment-898628367

https://github.com/mingrammer/diagrams/issues/358#issuecomment-716565948
https://github.com/mingrammer/diagrams/issues/358#issuecomment-723581944
https://github.com/mingrammer/diagrams/issues/358#issuecomment-723674931
